Cargo.toml

```diff
@@ -85,9 +85,9 @@ prometheus = "0.13"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 snafu = "0.8"
-surf-disco = "0.7"
+surf-disco = "0.6"
```

Comment:
why do we want to downgrade?

Reply:
We don't really, but we can't merge this into the sequencer with 0.7 until the sequencer is also on 0.7, and *that* can't happen until HotSHot is on 0.7

Reply:
right